packages/nimble-components/src/table/specs/table-column-specs/table-column-mapping-anchor.md

```diff
@@ -0,0 +1,197 @@
+# Mapping Anchor Table Column
+
+## Overview
+
+The `nimble-table-column-mapping-anchor` is a component that supports rendering specific number, boolean, or string values as an icon or a spinner, with an accompanying text anchor element. The mappings are defined by child elements of `nimble-mapping-icon`, `nimble-mapping-spinner`, or `nimble-mapping-empty`.
+
+### Background
+
+[Icon with anchor column](https://github.com/ni/nimble/issues/2531)
+
+### Features
+
+- Supported input:
+    - string
+    - number
+    - boolean
+- Supported output:
+    - Icon with text link
+    - Spinner with text link
+    - text link
+
+### Non-goals
+
+- Non-Nimble icon support
+- Arbitrary icon colors
+- Non-icon, non-spinner Nimble components
+- Specifying different text for an icon/spinner's label than the overall text of the mapping
+
+---
+
+## Design
+
+Below is an example of how these elements would be used within a `nimble-table`:
+
+```HTML
+<nimble-table>
+    <nimble-table-column-mapping-anchor field-name="status" key-type="string" label-field-name="status" href-field-name="status-url">
+        Status
+        <nimble-mapping-icon key="fail" icon="nimble-icon-xmark" severity="error" text="Failed"></nimble-mapping-icon>
+        <nimble-mapping-icon key="error" icon="nimble-icon-xmark" severity="error" text="Errored"></nimble-mapping-icon>
+        <nimble-mapping-icon key="pass" icon="nimble-icon-check" severity="success" text="Passed"></nimble-mapping-icon>
+        <nimble-mapping-spinner key="running" text="Running"></nimble-mapping-spinner>
+    </nimble-table-column-mapping-anchor>
+</nimble-table>
+```
+
+The notable distinction between this column type and the `nimble-table-column-mapping` is the `label-field-name` and `href-field-name` attributes, and the absence of supporting the `nimble-mapping-text` as one of its mapping types. The two new attributes are aligned with the `nimble-table-column-anchor` API, and provide the fields in the table record for the text and URL for the displayed link. The `nimble-mapping-text` mapping will not be supported as it seems unnecessary for the use-cases needed.
+
+See the [Mapping Table Column spec](./table-column-mapping.md#design), and the [Anchor Table Column spec](./table-column-anchor-hld.md#implementation--design) for more details related to the design.
+
+### API
+
+#### Enum column element:
+
+_Component Name_
+
+- `nimble-table-column-mapping-anchor`
+
+_Props/Attrs_
+
+##### Attributes inherited from TableColumnEnumBase
+
+- `field-name`: string
+- `key-type`: 'string' | 'number' | 'boolean'
+
+##### Attributes "copied" from TableColumnAnchor (intend to move to mixin)
+
+- `label-field-name`: string
+- `href-field-name`: string
+- `appearance`: string
+- `underline-hidden`: string
+- `hreflang`
+- `ping`
+- `referrerpolicy`
+- `rel`
+- `target`
+- `type`
+- `download`
```

Comment:
Might too much detail for HLD but was thinking:
- a mixin for the native anchor attributes, i.e. `hreflang`, `ping`, `target`, etc.
- a mixin for AnchorColumns which calls the above mixin and adds `label-filed-name`, `href-field-name`, `appearance`, `underline`

Reply:
Cool. I think I'll leave that out of this HLD. May want to discuss details of this idea out-of-band.